transports/http/src/main/java/org/mule/compatibility/transport/http/HttpMessageProcessTemplate.java

```diff
@@ -224,10 +224,11 @@ public void afterFailureProcessingFlow(MuleException exception) throws MuleExcep
             }
             catch (Exception e)
             {
-                logger.warn("Exception sending http response after error: " + e.getMessage());
+                final String errorMessage = "Exception sending http response after error: ";
+                logger.warn(errorMessage + e.getMessage());
                 if (logger.isDebugEnabled())
                 {
-                    logger.debug(e.toString());
+                    logger.debug(errorMessage, e);
```

Comment:
The ": " ending of the message shouldn't be logged in this case, only on the previous one.